src/material.cpp

```diff
@@ -1436,4 +1491,72 @@ openmc_extend_materials(int32_t n, int32_t* index_start, int32_t* index_end)
 
 extern "C" size_t n_materials() { return model::materials.size(); }
 
+//
+// cvmt functions 
+double 
+PolyProperty::evaluate(Position r) {
+  double results {0.0}; 
+  if(type_ == "zernike1d"){ 
+    results = evaluate_zernike1d(r);
+  } else if (type_ == "zernike"){
+    results = evaluate_zernike(r);
+   }
+  return results;
+}
+
+double 
+PolyProperty::evaluate_zernike1d(Position r) {
+  int i;
+  double rho;
+  int c_index;
+  double poly_val;
+  double property {0.0};
+  //
+  rho = std::sqrt(r.x * r.x + r.y * r.y) / coeffs_[0];
+  c_index = 2;
+  for( i=0; i <= order_; i=i+2){
+      calc_zn_rad(i, rho, &poly_val);
+      property += poly_val * coeffs_[c_index-1];
+      c_index = c_index + 1;
+  }
+  return property;
+}
+
+double 
+PolyProperty::evaluate_zernike(Position r) {
+  int i, j, k;
+  double rho;
+  double phi;
```

Comment:
Same thing here, declare these variables at the time they are needed so they are properly initialized